configure.ac

```diff
@@ -1954,6 +1967,11 @@ fi
 case $add_players in
   *omxplayer*)
       XB_ADD_PLAYER([OMXPLAYER], [omxplayer])
+      AC_DEFINE([TARGET_RASPBERRY_PI],[1],["Define to 1 for RASPBERRY_PI"])
```

Comment:
I still don't understand why you can't use --with-platform=raspberry-pi. It'd be trivial to add rpi2.
Is this some stupid distro policy?